enkit/outputs/commands.go

```diff
@@ -303,5 +303,24 @@ func NewShutdown(root *Root) *Shutdown {
 }
 
 func (c *Shutdown) Run(cmd *cobra.Command, args []string) error {
-	return fmt.Errorf("`enkit outputs Shutdown` is unimplemented")
+	bbOpts := bbexec.NewClientOptions(
+		c.root.Log,
+		0, /* tunnel port does not matter in this case */
+		c.root.OutputsRoot,
+	)
+	bbClient, err := bbexec.MaybeStartClient(bbOpts)
```

Comment:
This merits a comment - maybe `// MaybeStartClient is used here to bind a client handle to an existing process, so that we can kill it. It may start a process that will be then killed quickly, which is acceptable but not ideal.`